icecreamshop/icecreamshop/src/test/java/com/summerschool/icecreamshop/product/controller/ProductControllerTest.java

```diff
@@ -12,49 +17,24 @@
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
 import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import java.util.Optional;
+
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-import java.util.Optional;
-
 @RunWith(SpringRunner.class)
 @ExtendWith(SpringExtension.class)
 @WebMvcTest(ProductController.class)
 @AutoConfigureMockMvc
 class ProductControllerTest {
 
-    @MockBean
-    ProductService productService;
-
-    @Autowired
-    MockMvc mockMvc;
-
-    @Test
-    void givenIdWhenFindByIdThenStatusIsOk() throws Exception {
-
-        Long idProduct = 1L;
-        Product product = new Product();
-        product.setId(Long.valueOf(idProduct));
-
-        Mockito.when(productService.findById(idProduct)).thenReturn(Optional.of(product));
-
-        mockMvc.perform(get("/products/"+idProduct))
-                .andExpect(status().isOk())
-                .andExpect(jsonPath("id", Matchers.is(idProduct)));
-    }
-
-    @Test
-    void givenIdWhenFindByIdThenStatusIsNotFound() throws Exception {
-
-        Long idProduct = -100L;
-
```

Comment:
hello, pare ca testele au fost sterse, e ceva in neregula cu ele? Cel mai bine ar fi fost sa fie pastrate. Ca best practice, orice bucata de cod ar trebui sa aibe test/e care sa confirme ca totul merge ok.